new smart contracts/contracts/Collection.sol

```diff
@@ -1,25 +1,64 @@
 /// SPDX-License-Identifier: BUSL-1.1
 /// Copyright © 2022 Fragcolor Pte. Ltd.
 
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.13;
 
 import {ERC721A} from "erc721a/contracts/ERC721A.sol";
 import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
-import {MerkleProof} from "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
-import {Clone} from "clones-with-immutable-args/Clone.sol";
-
+import {FragnovaBaseUri} from "./FragnovaBaseUri.sol";
+import {UpdatableOperatorFilterer} from "operator-filter-registry/src/UpdatableOperatorFilterer.sol";
+import {ERC2981} from "@openzeppelin/contracts/token/common/ERC2981.sol";
 import "hardhat/console.sol";
 
-contract Collection is Clone, Ownable, ERC721A {
+address constant OPENSEA_REGISTRY = address(
+    0x000000000000AAeB6D7670E522A718067333cd4E
+);
+address constant DEFAULT_OPENSEA_SUBSCRIPTION = address(
+    0x3cc6CddA760b79bAfa08dF41ECFA224f810dCeB6
+);
+
+contract Collection is
+    Ownable,
+    ERC721A("", ""),
+    UpdatableOperatorFilterer(
+        OPENSEA_REGISTRY,
```

Comment:
Are those address valid on every eth chains? If not we need a better way as we might not deploy on ETH

Reply:
Yup they are: https://github.com/ProjectOpenSea/operator-filter-registry#deployments

But it doesn't matter since the **constructor will never be called**. 

The owner will manually have to call `register()` / `unregister()` on the OpenSea Filter Registry Smart Contract (look at the `@notice` I put for `ClonedCollection.sol` for a more detailed explanation). 

Although - we can add helper functions on the Collection contract itself - but I don't see the need since the owner can manually do it ?

Reply:
hmm maybe in our calldata constructor factory we should add an option to auto register?

Reply:
> Styling with Markdown is supportedCommentCancel
> Resolve conversation

done. please resolve this conversation if it's okay